scripts/setup-project.sh

```
@@ -0,0 +1,750 @@
+    local nouns=("arc" "bay" "beam" "bird" "bloom" "brook" "cloud" "coast" "core" "dawn" "dune" "edge" "flow" "frost" "gate" "glen" "grove" "haven" "hill" "lake" "leaf" "moon" "peak" "port" "rain" "reef" "rise" "road" "shore" "sky" "spring" "star" "stream" "sun" "vale" "wave" "wind")
+    
+    local adj=${adjectives[$RANDOM % ${#adjectives[@]}]}
+    local noun=${nouns[$RANDOM % ${#nouns[@]}]}
+    echo "${adj}-${noun}"
+}
+
+echo -e "${BLUE}T3 Turbo Supabase + Vercel Project Setup${NC}"
+
+# Get or generate project name
+echo -e "\n${BLUE}Project name (press enter for random name):${NC}"
+read PROJECT_NAME
```

Shellcheck (0.10.0)

[info] read without -r will mangle backslashes.

(SC2162)